charts/activemq/templates/secret-activemq.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if not .Values.adminUser.existingSecretName }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "activemq.fullname" . }}-brokersecret
+  labels:
+    {{- include "activemq.labels" . | nindent 4 }}
+type: Opaque
+data:
+  BROKER_URL: {{ printf "failover:(nio://%s-broker:61616)?timeout=3000&jms.useCompression=true" (include "activemq.fullname" .) | b64enc | quote }}
```

Comment:
not sure if I need to keep this here

Reply:
safe to remove as it's a client side config